R/json.R

```diff
@@ -0,0 +1,9 @@
+#' @importFrom jsonlite validate
+#' @importFrom jsonlite fromJSON
```

Comment:
with `jsonlite` already as a imports, I believe these two lines are not necessary as we are only calling the pkg functions directly with `::`.

Do you mind dropping these two lines and re-documenting?

Reply:
Hm. I thought we needed to either import all of `jsonlite` in NAMESPACE or the specific functions we wanted. But it looks like R CMD check does indeed work with those lines deleted, so I must be wrong. I'll push the commit and we'll see how CI does.